.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+# Inlcuding the tests with ubuntu to get actual test coverage. The OSX builds skip seccomp tests.
+name: test-coverage
+on: [push, pull_request]
+jobs:
+  validate:
+    name: Validate 
+    runs-on: ubuntu-latest
+    # Pull requests from the same repository won't trigger this checks as they were already triggered by the push
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
We could also run it on every pull request as well as filtering `push` events to the `master` branch. Then we could avoid both lines I guess.

Reply:
Sorry, I don't get it.